src/client/containers/fetchMethod/fetch.js

```diff
@@ -24,6 +24,9 @@ export async function fetchFromDb(endpoint, fetchMethod, postBody = {}) {
   if (!response.ok) {
     throw new ApiError(response.statusText, response.status);
   }
-  const dbData = await response.json();
-  return dbData;
+  if (fetchMethod === 'get') {
```

Comment:
You can check the headers to see what it returns
` response.headers.get("content-type");`

Reply:
This returns null If is post, put, or delete. if is get it returns "application/json; charset=utf-8"